docs/migration/5_0.md

```diff
@@ -654,3 +654,7 @@ Due to i18next migration, certain set of keys have been migrated (from `_plural`
 #### Template Changes
 
 - Changed `<ng-container *ngIf="(typeSelected$ | async) && !(isUpdating$ | async); else loading">` to `<ng-container *ngIf="!!paymentTypes.length && (typeSelected$ | async) && !(isUpdating$ | async); else loading">`
+
+### ProductCarouselComponent
+
+- Template code for carousel item has been moved to a reusable component - `ProductCarouselItemComponent`.
```

Comment:
Can you please mention the new tag `<cx-product-carousel-item>` that will appear in the DOM.  Thanks!